### PR TITLE
Added maven plugin

### PR DESCRIPTION
**Pull request by jitpack-io**
So that its easy to build on jitpack.io and get spotify-web-api-android as a maven/gradle dependency.

This how it looks in our fork:
https://jitpack.io/#jitpack-io/spotify-web-api-android/0.2

### CODE REVIEW

**Comment by SoftwareWarlock** [comment]
Would be really nice to have this, or at least a new release that jitpack can use.

**Comment by jitpack-io** [comment]
@SoftwareWarlock indeed. Looks like the PR needs re-basing now. Happy to rebase if @kaaes is interested

**Comment by kaaes** [comment]
@jitpack-io let's give it a try!

**Comment by jitpack-io** [comment]
@kaaes Great! Updated the PR and can confirm it working on JitPack.io

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ buildscript {
         classpath "com.android.tools.build:gradle:${ANDROID_GRADLE_PLUGIN_VERSION}"
         classpath "org.robolectric:robolectric-gradle-plugin:${ROBOLECTRIC_GRADLE_PLUGIN_VERSION}"
         classpath "com.jakewharton.sdkmanager:gradle-plugin:${SDKMANAGER_GRADLE_PLUGIN_VERSION}"
-
+	classpath "com.github.dcendents:android-maven-gradle-plugin:1.3"
```
Comment (d-kunin) [inline build.gradle]: I guess you have used tabs instead of spaces here, that is why diff looks not aligned.
Please use spaces instead.

Comment (jitpack-io) [inline build.gradle]: Good catch. Fixed

**Comment by kaaes** [comment]
cool! @jitpack-io out of curiosity - if we add another module with its own artifact to this project (for example for the Echo Nest #58) will it be possible to fetch both from jitpack?

**Comment by jitpack-io** [comment]
@kaaes Absolutely. Multi module builds are supported https://jitpack.io/docs/BUILDING/#multi-module-projects

**Comment by kaaes** [comment]
@jitpack-io awesome :) I'll update the docs with this change

**Pull request merged**